src/Glimpse.Common/Reflection/DefaultTypeActivator.cs

```diff
@@ -31,21 +31,29 @@ public T CreateInstance<T>(params object[] parameters)
 
         public IEnumerable<object> CreateInstances(IEnumerable<TypeInfo> types)
         {
-            var activated = types.Select(t => CreateInstance(t));
+            var activated = types.Select(t => CreateInstance(t)).Where(o => o != null);
```

Comment:
For perf I wonder if we should just change this to a for loop and drop the expression. Save iterating over the Enumeration twice.